environment.yml

```diff
@@ -2,4 +2,4 @@ name: analysis
 channels:
   - conda-forge
 dependencies:
-  - root=6.16.00
+  - root=6.28.00
```

Comment:
Works fine on REANA using official ROOT container images too.

The necessary change to `reana.yaml` is:

```diff
diff --git a/reana.yaml b/reana.yaml
index 1786fa1..603fe67 100644
--- a/reana.yaml
+++ b/reana.yaml
@@ -7,11 +7,11 @@ workflow:
   specification:
     steps:
       - name: compile
-        environment: reanahub/reana-env-root6:6.18.04
+        environment: rootproject/root:6.28.00-ubuntu22.04
         commands:
           - g++ -v -O3 -o dimuonSpectrum dimuonSpectrum.C `root-config --cflags --libs`
       - name: run
-        environment: reanahub/reana-env-root6:6.18.04
+        environment: rootproject/root:6.28.00-ubuntu22.04
         commands:
           - ./dimuonSpectrum
 outputs:
``` 

The result:

![dimuonspectrum](https://github.com/cms-opendata-analyses/DimuonSpectrumNanoAODOutreachAnalysis/assets/517546/fc982f74-a905-4b28-bb54-c61724a5b1d3)

Reply:
OK, thanks, updated!